plugins/__init__.py

```diff
@@ -1313,6 +1314,27 @@ def format(__string: "string", *args: "any", **kwargs: "any") -> "string":
 try:
     from inmanta.references import Reference, reference
 
+    @reference("std::IntReference")
+    class IntReference(Reference[int]):
+        """A reference that converts a reference value to an int"""
+
+        def __init__(self, value: object | Reference[object]) -> None:
```

Comment:
Can't we type this as follows ?
```suggestion
        def __init__(self, value: int | Reference[int]) -> None:
```

Reply:
I don't think so, the main use case is to put a FactReference[str] as input so the typing won't be correct